### PR TITLE
Sprint report for sprint 22

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: _posts/development/2017-08-18-sprint-report-22.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+   <li>UpdatePackageMeta  job is now covered by 100% (<a href='https://github.com/openSUSE/open-build-service/pull/3564'>PR#3564</a>)</li>
+   <li>Event mailer is now covered by 100% (<a href='https://github.com/openSUSE/open-build-service/pull/3554'>PR#3554</a>)</li>
+ </ul>
+ 
+ <p>
+ Beside migrating tests to the new test suite, we also fixed several flickering tests (tests which sometimes report a false negative). 
+ Have a look at <a href="https://github.com/openSUSE/open-build-service/pull/3628">#3628</a> and <a href="https://github.com/openSUSE/open-build-service/pull/3626">#3626</a>.
+ </p>
+ 
+ <h2>Views</h2>
```
Comment (hennevogel) [inline _posts/development/2017-08-18-sprint-report-22.md]: These are bugfixes too right? Your readers don't care about Views...

File: _posts/development/2017-08-18-sprint-report-22.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+   <li>
+      Keep asset precompilation in our package build like it is right now and document in our deployment wiki that we have to ensure there are no additional files in the /public/assets directory
+   </li>
+</ul>
+</p>
+<p>
+   In the end we decided for option 3 and <a href='https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org/34c35d60e10a368511b2bf9ed2dc718b38395f2d'>documented</a> what deployers should do to avoid this from breaking things in our wiki.
+</p>
+<p>
+   The reason we didn't decide for one of the other solutions was that this would have made our package installation more complex and would have affected other people that want to host their own OBS to much.
```
Comment (eduardoj) [inline _posts/development/2017-08-18-sprint-report-22.md]: Typo: `... OBS to much.` should be `... OBS too much.`.

File: _posts/development/2017-08-18-sprint-report-22.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+<h3>Remove excluded repositories from rpmlint dropdown</h3>
+<p>
+   Do you need to review a lot of submit requests?
+   Are you annoyed to go over all repository architecture combinations for the rpmlint result to check that no errors will get introduced?
+</p>
+<p>
+   We did several changes to make this workflow easier for maintainers.
+   First of all we dropped now excluded and the images repositories (e.g. when you build a debian package against openSUSE) from this list as these don't have rpmlint results.
+   Furthermore we changed the ordering of the architectures to reverse to favor the x86_64 architecture.
+   You can read about it in <a href="https://github.com/openSUSE/open-build-service/issues/3183">#3186</a>, <a href="https://github.com/openSUSE/open-build-service/issues/3608">#3608</a> and <a href="https://github.com/openSUSE/open-build-service/issues/3622">#3622</a>.
```
Comment (eduardoj) [inline _posts/development/2017-08-18-sprint-report-22.md]: Issue should be 3186 in the link.

**Comment by ChrisBr** [comment]
@eduardoj thanks, fixed!
